src/main/java/de/retest/util/StringSimilarity.java

```diff
@@ -86,4 +89,8 @@ public static double simpleSimilarity( final String s0, final String s1 ) {
 		return 0.0;
 	}
 
+	private static String removeBrackets( final String path ) {
+		return path.replaceAll( "]", "" ).replaceAll( "\\[", "" );
```

Comment:
Why not use one regex for both? @martin-v is our regex guru… ;-)

Reply:
`path.replaceAll( "[\[\]]", "" )` should do the jobs

Reply:
As a tip, for the next time play with https://regex101.com/r/6QEbLC/1/